ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml

```diff
@@ -89,6 +89,7 @@ tests:
                   --e2e.platform="KubeVirt" \
                   --e2e.latest-release-image=${OCP_IMAGE_LATEST} \
                   --e2e.previous-release-image=${OCP_IMAGE_PREVIOUS} \
+                  --e2e.control-plane-operator-image=${HYPERSHIFT_RELEASE_LATEST} \
```

Comment:
@davidvossel the CPO image comes from HC.release.image which is dictated by `--e2e.latest-release-image=${OCP_IMAGE_LATEST}` which should be the latest CI ephemeral payload containing your PR changes.
How does this change help? fwiw this would enable a disassociation of the CPO image from the components it runs, and so you'd be testing something  you'll never run in a real env, control-plane-operator-image is meant to be used by devs.

Reply:
@enxebre The kubevirt provider is having issues with 4.12 because CNV itself can't currently be deployed on 4.12. 

This has caused us to temporarily create our own prow config that is unique to KubeVirt where[ `latest` is 4.11 instead of 4.12](https://github.com/openshift/release/blob/61b4dbc85950ae0a28a549eb4de2cc6fe8f4d8e7/ci-operator/config/openshift/hypershift/openshift-hypershift-main__kubevirt.yaml#L45)

What this means is, for kubevirt we're deploying the latest 4.11 infra/mgmt cluster and using a 4.12 guest payload. The `include_built_images: true` can only be applied to the `latest` integration, which in our case is 4.11. And that's not the payload we're using for the guest.

example... latest is 4.11, and we have a 4.12 integration as well that's used for the guest payload.

```
  latest:
    integration:
      include_built_images: true
      name: "4.11"
      namespace: ocp
  v412:
    integration:
      name: "4.12"
      namespace: ocp
```

The the OCP_IMAGE_LATEST is overwritten to be 4.12 instead of latest (which is 4.11 for us)

```
      - env: OCP_IMAGE_LATEST
        name: release:v412
```

Now, if we don't use `--e2e.control-plane-operator-image=${HYPERSHIFT_RELEASE_LATEST}`, then we won't get the latest HCCO from the PR in our presubmit, Instead we'll get whatever is in the 4.12 release payload.